articles/modules/ROOT/pages/setting-up-ssl-with-docker.adoc

```diff
@@ -10,10 +10,141 @@ Neo4j 3.2 added a Unified SSL Framework to setup secure connections for Bolt, HT
 framework can be found at:  https://neo4j.com/docs/operations-manual/current/security/ssl-framework/
 
 Setting up secure Bolt and HTTPS communications when running Neo4j within a Docker Container requires some specific steps and 
-configuration settings:
+configuration settings.
 
 The below steps assume you've followed the process outlined in the above documentation link and have created your public cert and
-private key.
+private key. 
+
+CAUTION: The steps listed below are for example purposes only. You should make sure these are suitable for your requirements before deploying into any environment. 
```

Comment:
Suggest replacing "You/Your" with e.g. "One should make sure" or "it must be ensured these are suitable for given requirements"